ui/push-health/CommitHistory.jsx

```diff
@@ -55,93 +61,121 @@ class CommitHistory extends React.PureComponent {
+              <span>{authorEmail}</span>
+            </div>
           </div>
-        </div>
-        {revisions.length <= 5 || showAllRevisions ? (
-          <RevisionList
-            revision={revision}
-            revisions={revisions.slice(0, 20)}
-            revisionCount={revisionCount}
-            repo={currentRepo}
-          />
-        ) : (
-          <span>
-            <RevisionList
-              revision={revision}
-              revisions={revisions.slice(0, 5)}
-              revisionCount={revisionCount}
-              repo={currentRepo}
-            />
-            <Button
-              outline
-              color="darker-secondary"
-              onClick={() =>
-                this.setState({ showAllRevisions: !showAllRevisions })
-              }
-            >
-              Show more...
-            </Button>
-          </span>
-        )}
-        <div className="mt-4">
-          Parent Push:
-          <span className="ml-2">
-            {!exactMatch && (
-              <div>
-                <Alert color="warning" className="m-3 font-italics">
-                  Warning: Could not find an exact match parent Push in
-                  Treeherder.
-                </Alert>
-                {id && <div>Closest match: </div>}
-              </div>
-            )}
-            <span
-              className="mb-2"
-              onMouseEnter={() => this.showClipboard(true)}
-              onMouseLeave={() => this.showClipboard(false)}
-            >
-              <Clipboard
-                description="full hash"
-                text={parentSha}
-                visible={clipboardVisible}
-              />
+          <div className="text-secondary">
+            Push{' '}
+            <span className="font-weight-bold">
               <a
-                href={parentLinkUrl}
+                href={revisionPushFilterUrl}
                 target="_blank"
                 rel="noopener noreferrer"
-                title="Open this push"
-                data-testid="parent-commit-sha"
-                className="mr-1 text-monospace commit-sha"
               >
-                {parentPushRevision || parentSha}
+                {revision.substring(0, 17)}...
               </a>
-              {exactMatch && (
-                <PushHealthStatus
-                  revision={parentPushRevision}
-                  repoName={parentRepository.name}
-                  jobCounts={jobCounts}
+            </span>{' '}
+            on{' '}
+            <span className="text-capitalize font-weight-bold">
+              {currentRepo.name}
+            </span>
+          </div>
+        </div>
+        <div className="commit-area mt-2 pl-2 text-secondary">
+          <span className="font-weight-bold">
+            <FontAwesomeIcon
+              onClick={this.toggleDetails}
+              icon={expandIcon}
+              title={expandTitle}
+              aria-label={expandTitle}
+              alt=""
+            />{' '}
```

Comment:
replace the `{' '}` with `mr-1` on the icon.

Also, let's make the click target be the whole text, not just the icon.  It should also have a `cursor: pointer` to make it more obvious it is doing something.  In this case, you might change the `<span` on line 100 to be a `reactstrap` `<Button>`.  If you add the `outline` param to the `Button` as well as the class of `border-0` then it will work nicely and have the pointer, but won't have the button background.  There are a few examples of me doing this in other areas.